src/modules/Header/components/Avatar.tsx

```diff
@@ -8,18 +8,18 @@ import { Link, useNavigate } from '@tanstack/react-router';
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
-import { queryClient } from '@/lib/react-query';
 import { notify } from '@/utils/notify';
 
 import { useLogout } from '../api/logout';
 
-export const Avatar = () => {
+type AvatarProps = {
+  username: string;
```

Comment:
```suggestion
  username: User['user'];
```

you should always avoid hardcoding types from schema, like username in this case, you should use type from schema, in this case it is not super important cause username type won't change but imagine if you had id of type string, and it changes to number, in this case you need to go through all app change type to number

Reply:
Done